agent/src/softwarecontaineragentadaptor.h

```diff
@@ -0,0 +1,56 @@
+#ifndef SOFTWARECONTAINERAGENTADAPTOR_H
+#define SOFTWARECONTAINERAGENTADAPTOR_H
+
+#include "softwarecontaineragent.h"
+
+namespace softwarecontainer {
+
+class SoftwareContainerAgentAdaptor :
+    public com::pelagicore::SoftwareContainerAgent_adaptor
```

Comment:
It's not needed but is not part of this story. I'd suggest you do it in your cleanup story ;)